return_interfacial_hbond_coordinates.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def extract_lists(half1_dict, half2_dict):
+    '''Returns 2 lists of tuples containing only the apex atom coordinates for both dictionaries'''
+    half1_list = []
+    half2_list = []
+    
+    # iterate over each key-value pair in half1_dict
+    for value in half1_dict.values(): # where did we define half1_dict?
```

Comment:
half1_dict is a parameter passed to the function. You cannot call extract_lists without passing half1_dict to the function as the first parameter